app/src/main/res/values-uk/strings.xml

```diff
@@ -286,7 +285,7 @@
     <string name="enter_card_id">Введіть ID або текст на вашій картці</string>
     <string name="addWithoutBarcode">Додати картку без штрих-коду</string>
     <string name="field_must_not_be_empty">Поле вводу не повинно бути порожнім</string>
-    <string name="app_name">Catima</string>
+    <string name="app_name">Катіма</string>
```

Comment:
Asked if translating product names is normal in Ukraine: https://hosted.weblate.org/translate/catima/catima/uk/?&offset=1#comments

Reply:
Haven't heard back from the translator yet. @Korb, you were Ukrainian IIRC, right? Do you know if this would be a logical translation or not?

Reply:
@TheLastProject, the rule of mandatory cyrillization of names is legally applicable only to official documentation (texts of laws, accompanying documents of declared goods, names of legal entities, and many others), but not to trademarks, trade marks, and, especially, free software.

I believe that the name of the application should be written everywhere only in its original form, in Latin, without translation or transliteration, regardless of the language of the application interface.

Reply:
Thanks for your quick response! I reverted their change for now and made #2118 to at some point automate checking this (I know in Hindi it apparently is quite normal to transliterate trademarks and the likes, so an allowlist will be necessary, but I'll remember in Ukrainian it's better not to cyrillize)

Reply:
> I'll remember in Ukrainian it's better not to cyrillize)

And at ruzzian too.